lib/matplotlib/axes/_base.py

```diff
@@ -3995,9 +3995,10 @@ def _get_view(self):
             default implementation saves the view limits. You *must* implement
             :meth:`_set_view` if you implement this method.
         """
-        xmin, xmax = self.get_xlim()
-        ymin, ymax = self.get_ylim()
-        return xmin, xmax, ymin, ymax
+        return {
+            "xlim": self.get_xlim(), "autoscalex_on": self.get_autoscalex_on(),
+            "ylim": self.get_ylim(), "autoscaley_on": self.get_autoscaley_on(),
```

Comment:
Slight preference for separate lines:
```suggestion
            "xlim": self.get_xlim(),
            "ylim": self.get_ylim(),
            "autoscalex_on": self.get_autoscalex_on(),
            "autoscaley_on": self.get_autoscaley_on(),
```

Reply:
I have a slight preference for keeping them together because they actually logically belong together.

Reply:
Ok